### DISCUSSION TITLE
How can I make a DELETE request from an edge template

### DISCUSSION THREAD

**Discussion by silentworks** [adonisjs/core#1572] [Help] [answered]
I have a DELETE route and want to make a request to it from the edge template but when I try using `{{ route('PostsController.destroy', { params: { id: post.id }, qs: { _method: 'delete'} }) }}`, the `_method` doesn't seem to work anymore, what is the workaround for this without the need for JavaScript?

**Comment by thetutlage** [comment, marked as answer]
Make sure `allowMethodSpoofing` is set to true inside `config/app.ts` file

**Reply by silentworks** [reply]
Thank you very much, that worked.
